components/raftstore/src/store/fsm/apply.rs

```diff
@@ -3200,24 +3211,35 @@ where
         fail_point!("on_handle_apply_2", self.delegate.id() == 2, |_| {});
         fail_point!("on_handle_apply", |_| {});
 
-        if apply.entries.range.is_empty() || self.delegate.pending_remove || self.delegate.stopped {
+        if self.delegate.pending_remove || self.delegate.stopped {
```

Comment:
Why remove this?

Reply:
Because `apply.entries` is a `SmallVec` now and it should be always non-empty.